src/plugins/monitor/src/monMain.c

```diff
@@ -719,13 +719,14 @@ static int32_t monBuildMnodesTotalSql(char *sql) {
 
 static int32_t monGetVgroupsTotalStats(char *dbName, int32_t *totalVgroups,
                                            int32_t *totalVgroupsAlive) {
-  char subsql[TSDB_DB_NAME_LEN + 14];
+  int  bufLen = TSDB_DB_NAME_LEN + 16;
+  char subsql[bufLen];
   memset(subsql, 0, sizeof(subsql));
-  snprintf(subsql, TSDB_DB_NAME_LEN + 13, "show %s.vgroups", dbName);
+  snprintf(subsql, bufLen - 1, "show `%s`.vgroups", dbName);
```

Comment:
The database names used in Taos shell, should be backwards quoted from now on, in order to treat them case sensitively or to support them to be Chinese characters. Otherwise, a syntax error or a wrong DB accessing might happen.